gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -308,12 +292,14 @@ public MetricsRegistry getRegistry() {
   }
 
   /**
-   * Look up a metric from both the registered set and the lighter weight stream entries.
+   * Look up a metric from both the registered set and the lighter weight stream entries. If the
+   * metric is status statistic, then look from the statusStatistic instance's registry
    *
    * @param name metric name
    * @return the metric or null
    */
   public MutableMetric lookupMetric(String name) {
+    if (isStatusStatistic(name)) return getInstanceOfStatusStatisticListener().lookupMetric(name);
```

Comment:
We avoid using single line if statements since they are error prone. Please change.